config.go

```diff
@@ -98,6 +99,10 @@ const (
 	// HTLCs on our channels.
 	minTimeLockDelta = routing.MinCLTVDelta
 
+	// MaxTimeLockDelta is the maximum CLTV delta that can be applied to
+	// forwarded HTLCs.
+	MaxTimeLockDelta = math.MaxUint16
```

Comment:
i think this should be a constant in the `routing` package where `MinCLTVDelta` is also currently defined. 

Then we can replace [this](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/invoicesrpc/addinvoice.go#L351) check with that constant too.